contracts/truefi2/TrueFiPool2.sol

```diff
@@ -103,9 +103,10 @@ contract TrueFiPool2 is ITrueFiPool2, IPauseableContract, ERC20, UpgradeableClai
         ERC20 _token,
         ITrueLender2 _lender,
         ISAFU _safu,
-        address __owner
+        address __owner,
+        string memory name
```

Comment:
I think argument name "name" is misleading. I understand that the pool will be called something like "TrueFi Alameda USDC". I have no idea what it should be called. Maybe we need to have separate initializers for public and private pools? This one will stay same and private initializer will take full name and symbol?